### ISSUE TITLE
NURBS support

### ISSUE THREAD

**Issue by kmokstad** [TheBB/IFEM-to-VT#5]
According to @akva2, this is not supported yet. Can it be done?

**Comment by kmokstad** [comment]
And a README.md with installation instructions would be helpful

**Comment by VikingScientist** [comment]
[This code line](https://github.com/TheBB/IFEM-to-VT/blob/9c3789840cc754bb018847db5b1b13aabcc91beb/ifem_to_vt/reader.py#L76) seem to suggest at least partial support for rational splines

**Comment by akva2** [comment]
well it does not work...

**Comment by VikingScientist** [comment]
So this issue is only relevant for LR-splines since NURBS seem to work as intended for tensor surface- and volume splines

**Comment by akva2** [comment]
no, there are also issues with NURBS without LR. i sent eivind a file showcasing the issue at some point.

**Comment by TheBB** [comment]
There are non-LR test files already in the repo that are rational. The ones I got sent indeed do not work though, so the jury is out at the moment as to why.

For rational LR I'll write a Python-side wrapper in https://github.com/TheBB/lrsplines-python
